src/converters/lintConfigs/rules/convertRules.test.ts

```diff
@@ -107,41 +107,118 @@ describe("convertRules", () => {
         expect(ruleEquivalents).toEqual(new Map([["tslint-rule-a", ["eslint-rule-a"]]]));
     });
 
-    it("reports a failure when two outputs exist for a converted rule without a merger", () => {
-        // Arrange
-        const conversionResult = {
-            rules: [
-                {
-                    ruleName: "eslint-rule-a",
-                },
-                {
-                    ruleName: "eslint-rule-a",
-                },
-            ],
-        };
-        const { tslintRule, converters, mergers } = setupConversionEnvironment({
-            conversionResult,
+    describe("runs without an argument merger when both rules have the same arguments", () => {
+        [
+            { ruleArguments: undefined, testCase: "no arguments" },
+            { ruleArguments: [5], testCase: "number argument" },
+            { ruleArguments: ["allow-something"], testCase: "string argument" },
+            { ruleArguments: [{ max: 50 }], testCase: "object argument" },
+            { ruleArguments: [5, "allow-something", { max: 50 }], testCase: "multiple arguments" },
+        ].forEach(({ ruleArguments, testCase }) => {
+            it(testCase, () => {
```

Comment:
I think this is what you're looking for ✨ : https://jestjs.io/docs/api#testeachtablename-fn-timeout

```ts
        [[5], "no arguments"],
        // ...
    ],
    test('%s', (ruleArguments)) => {
```

Though, I think we can safely reduce this down to just the `undefined` case and the last, complex one. Since it uses `isEqual` from Lodash we don't particularly need to test the equality checking logic. Thoughts?

Reply:
I mainly use `jasmine` so I didn't know of this feature. I'll refactor these parametrized tests.